CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+* Add new `file` key to the `build` map. Equivalent of `docker build --file=<file>`.
```

Comment:
Can you rebase that on master so that it is below the "Unreleased" header? Thanks!